### PR TITLE
RSC can see list of jobs in region

### PR DESCRIPTION
**Pull request by Prozak8**
### PT Link
 https://www.pivotaltracker.com/story/show/161644415

### Included in PR

- Created job model
- Rewrote html to correctly display jobs
- Created Job controller 

### Screenshots

screenshot 2018-11-02 at 15 21 53

### CODE REVIEW

File: spec/models/job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
```
Comment (rupertlion) [inline spec/models/job_spec.rb]: i think you can remove this

File: features/rsc_can_see_list_of_jobs.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@javascript
+Feature: RSC Dashboard
```
Comment (rupertlion) [inline features/rsc_can_see_list_of_jobs.feature]: can we put the actual feature name here?

File: db/seeds.rb [hunk 3]
```diff
@@ -6,3 +6,8 @@
 Hospital.create!(name: "Stockholms Sjukhem")
 Hospital.create!(name: "GHP Arytmi Center Stockholm")
 
+Job.create!(name: "Doctor")
```
Comment (rupertlion) [inline db/seeds.rb:10]: we should do a Job.destroy_all a the top of the file

File: app/views/staffcos/index.html.haml [hunk 4]
```diff
@@ -4,9 +4,9 @@
       %h1 Staffing Companies
     #list-item.flex
     - @staffcos.each do |staffcos|
-      #hosp-name
+      #form-name
         %p= staffcos.name
-      #hosp-button
+      #form-button
         %button.button Remove
```
Comment (rupertlion) [inline app/views/staffcos/index.html.haml:10]: do we need button.button? I wonder if just %button will work?

Comment (RhonGabriel) [reply app/views/staffcos/index.html.haml:10]: the second "button" is the name of its styling

**Review by angiecstn — APPROVED** [review]

**Pull request merged**